### PR TITLE
fix(header.component): add static header links instead of content loader

### PR DESCRIPTION
**Pull request by TrygveUrdahl**
# PR Checklist
https://elvia.atlassian.net/wiki/spaces/TEAMATOM/pages/10427498683/Review+prosess

- [ ] Bumpet package?
- [ ] Updated changelog?
- [ ] Correct date in changelog?

## Describe PR briefly:
https://elvia.atlassian.net/browse/LEGO-3439

Added static main menu links. When the CMS-service responds with the CMS-entry, it is replaced (so if the main menu changes in the CMS, it will be updated here to after a little delay)

### CODE REVIEW

File: packages/web/src/app/shell/header/header.component.ts [hunk 1]
```diff
@@ -41,8 +41,21 @@ type MenuType = 'search' | 'mobileMenu' | null;
...
+      {
+        title: 'Components',
+        path: '/components',
+        entry_id: '"6RHCZ59jMvJZjcCMMsjvKk"',
+        entry: undefined as any,
+      },
+      { title: 'Patterns', path: '/patterns', entry_id: '"5oUsPjHfF6bNDHqub1EaFl"', entry: undefined as any },
+      { title: 'Tools', path: '/tools', entry_id: '"6zwtCTRgovdJkF91Fkh4jN"', entry: undefined as any },
+    ],
+  };
```
Comment (TomSchrier) [inline packages/web/src/app/shell/header/header.component.ts:58]: Er det greit at disse er public mtp. open source? 😄

**Review by TomSchrier — APPROVED** [review]
En mini-kommentar, ellers LGTM

File: packages/web/src/app/shell/header/header.component.ts (hunk 1 above)
Comment (TrygveUrdahl) [reply packages/web/src/app/shell/header/header.component.ts:58]: Jada, null stress det!

Comment (TrygveUrdahl) [reply packages/web/src/app/shell/header/header.component.ts:58]: Alt dette kan man uansett se i Network-tab når requestene sendes

**Pull request merged**